initial-promises/node-server/inventory/1.0/fusionAgent.cf

```diff
@@ -413,34 +413,20 @@ bundle agent addInformationsToInventory
     any::
       "uuiddefined" expression => isvariable("RUDDERUUID");
   files:
-    windows::
-      "C:/Progra~1/Rudder/var/tmp/inventory/.*.ocs"
-    #"${g.esc_rudder_var_tmp}\inventory\.*.ocs"
-        edit_line => add_information_to_inventory(${RUDDERUUID}, ${CFKEY}, ${USER}, ${polserv_uuid}),
+    uuiddefined::
```

Comment:
I factored the code

The line calles were exactlty the same between windows and not windows ...